packages/components/src/components/primitives/Button/types.ts

```diff
@@ -0,0 +1,9 @@
+import { ButtonProps as ChakraButtonProps } from '@chakra-ui/react';
+import { Sizes } from '../../../theme/types';
+
+export type Variant = 'primary' | 'secondary' | 'tertiary';
```

Comment:
this variants should be unified with the ones defined in the constants file, please follow the TextPairing implementation